docs/block-api.md

```diff
@@ -114,6 +114,34 @@ Sometimes a block could have aliases that help users discover it while searching
 keywords: [ __( 'image' ), __( 'photo' ), __( 'pics' ) ],
 ```
 
+#### Styles (optional)
+
+* **Type:** `Array`
+
+Block Styles allow providing alternative styles to block. It works by adding a className to the block’s wrapper. This className can be used to provide an alternative styling for the block if the style variation is selected.
+
+```js
+// Register block styles.
+styles: [
+	// Mark style as default.
+	{ 
+		name: 'default', 
+		label: __( 'Rounded' ), 
+		isDefault: true 
+	},
+	{ 
+		name: 'outline', 
+		label: __( 'Outline' ) 
+	},
+	{ 
+		name: 'squared', 
+		label: __( 'Squared' ) 
+	},
+],
+```
+
+Plugins and Themes can also register [custom block style](../docs/extensibility/extending-blocks/#block-style-variations) for exisiting blocks.
```

Comment:
It should be a valid GItHub link: ../docs/extensibility/extending-blocks.md#block-style-variations. It will replace ../docs/ with a proper path in the handbook and it will also replace .md with /. @pento added logic forvthat a few months back. It isn’t ideal but works.

Reply:
We have similar links and they are working on handbook and GitHub both.